sklearn/linear_model/_glm/tests/test_glm.py

```diff
@@ -419,13 +419,13 @@ def test_tweedie_regression_family(regression_data):
 
 
 @pytest.mark.parametrize(
-        'estimator, value',
-        [
-            (PoissonRegressor(), True),
-            (GammaRegressor(), True),
-            (TweedieRegressor(power=1.5), True),
-            (TweedieRegressor(power=0), False)
-        ],
+    'estimator, value',
+    [
+        (PoissonRegressor(), True),
+        (GammaRegressor(), True),
+        (TweedieRegressor(power=1.5), True),
+        (TweedieRegressor(power=0), False),
+    ],
```

Comment:
It's fine for this time, but please in the future, try to avoid committing formatting changes away from lines that are impacted by the diff.